### PR TITLE
[new release] routes (0.6.0)

### PR DESCRIPTION
**Pull request by anuragsoni**
Typed routing for OCaml applications

- Project page: <a href="https://github.com/anuragsoni/routes">https://github.com/anuragsoni/routes</a>
- Documentation: <a href="https://anuragsoni.github.io/routes/">https://anuragsoni.github.io/routes/</a>

##### CHANGES:

* Improve mdx test tules (anuragsoni/routes#73, @NathanReb)
* Use github actions instead of travis ci (anuragsoni/routes#70, anuragsoni/routes#72)
* Get human readable route pattern from a route (anuragsoni/routes#64, anuragsoni/routes#74)
* Allow writing custom path match patterns (anuragsoni/routes#76)

### CODE REVIEW

File: packages/routes/routes.0.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+doc: "https://anuragsoni.github.io/routes/"
+tags: [ "router" "http" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" { >= "1.1" }
```
Comment (mseri) [inline packages/routes/routes.0.6.0/opam]: Are you sure that `(using fmt 1.1)` works with dune 1.1 and does not require  1.5?

Comment (anuragsoni) [reply packages/routes/routes.0.6.0/opam]: I think so. My understanding was that the version needs to match the dune language version in the dune-project file. I’ll check the docs again to confirm

Comment (kit-ty-kate) [reply packages/routes/routes.0.6.0/opam]: Currently dune extensions do not check for a matching dune version number (see https://github.com/ocaml/dune/issues/2957) and according to https://github.com/ocaml/dune/pull/1821, fmt 1.1 has been added in dune.1.7.0. So I believe your package requires at least dune 1.7

Comment (anuragsoni) [reply packages/routes/routes.0.6.0/opam]: Thank you for sharing those links. I think if I need to change the dune version constraint, I’ll go over the dune docs to see if there are any other extension that I’d like to use. Or see if I stop using the fmt extension. I’ll close this pull request for now and revisit this once I’ve figured out which dune extensions I need.

**Pull request closed without merging**